src/components/NetworkDebuggerModal/index.js

```diff
@@ -20,6 +21,7 @@ const NetworkDebuggerModal = ({onPress}) => {
     setIsNetworkVIsible(true);
   };
 
+  console.log(isDebug);
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The `console.log(isDebug);` statement seems to be a debugging line that was left in the code. It's generally not a good practice to leave console log statements in production code as it can clutter the console output and potentially expose sensitive information.
```diff
-  console.log(isDebug);
```



<!-- This is an auto-generated comment by OSS CodeRabbit -->